version/version.go

```diff
@@ -54,7 +54,9 @@ func init() {
 	rawVersion = strings.TrimSpace(rawVersion)
 
 	PackerVersion = pluginVersion.NewRawVersion(rawVersion)
-	SemVer = PackerVersion.SemVer()
+	// Since we already check that rawVersion is valid in NewRawVersion,
+	// we can safely ignore the error here.
+	SemVer, _ = version.NewVersion(rawVersion)
```

Comment:
Do we plan to revert back to using PackerVersion after the fix of the bug mentioned in the PR description? if so it may be better to comment out line 57 instead of deleting it, and then have a comment above explaining what bug needs to be closed before we can revert to using this PackerVersion.SemVer

Reply:
We could maybe revert back someday, but this is not necessary imho, we can continue with that implementation instead since this is virtually the same thing, besides the fact that we'll run a regex for validation again instead of returning the version, so it's slightly more expensive this way, but it's only executed once per process, so we can ignore this safely I think.